source/VehicleCommander/afmchemlight.json

```diff
@@ -1,124 +0,0 @@
-{
```

Comment:
You're right. That whole section in README.md is now irrelevant. I removed it in ac83f4a0af7d36b44ea071bb428160335e9eb418 .